From diegocg@gmail.com Thu Sep 21 13:39:17 2006
Date: Thu, 21 Sep 2006 22:37:10 +0200
From: Diego Calleja <diegocg@gmail.com>
To: Adrian Bunk <bunk@stusta.de>
Cc: jeff@garzik.org, torvalds@osdl.org, akpm@osdl.org,
 alan@lxorguk.ukuu.org.uk, greg@kroah.com, linux-kernel@vger.kernel.org
Subject: HOWTO: bug report addition
Message-Id: <20060921223710.d7472801.diegocg@gmail.com>

I suspect that not many people is subscribed to the bugzilla mailing list,
not surprising since the URLs doesn't seem to be in the tree :)

After fixing my english, I wonder if the following patch could be applied...

Signed-off-by: Diego Calleja <diegocg@gmail.com>
Acked-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/HOWTO |   20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

--- gregkh-2.6.orig/Documentation/HOWTO
+++ gregkh-2.6/Documentation/HOWTO
@@ -395,6 +395,26 @@ bugme-janitor mailing list (every change
 
 
 
+Managing bug reports
+--------------------
+
+One of the best ways to put into practice your hacking skills is by fixing
+bugs reported by other people. Not only you will help to make the kernel
+more stable, you'll learn to fix real world problems and you will improve
+your skills, and other developers will be aware of your presence. Fixing
+bugs is one of the best ways to get merits among other developers, because
+not many people like wasting time fixing other people's bugs.
+
+To work in the already reported bug reports, go to http://bugzilla.kernel.org.
+If you want to be advised of the future bug reports, you can subscribe to the
+bugme-new mailing list (only new bug reports are mailed here) or to the
+bugme-janitor mailing list (every change in the bugzilla is mailed here)
+
+	http://lists.osdl.org/mailman/listinfo/bugme-new
+	http://lists.osdl.org/mailman/listinfo/bugme-janitors
+
+
+
 Mailing lists
 -------------
 
